tests/console/openvswitch_ssl.pm

```diff
@@ -51,9 +52,12 @@ sub run {
     assert_script_run 'wget --quiet ' . data_url("$fname");
     assert_script_run "tar jvfx $fname";
 
+    my $python = "";    # use default python
+    $python = "python3.11 " if is_tumbleweed;    # On tumbleweed use python3.11 bsc#1239175
+
     # Setup a simulated open-flow controller with POX
     type_string
-"pox/./pox.py openflow.of_01 --port=6634 --private-key=/etc/openvswitch/server-key.pem --certificate=/etc/openvswitch/server-cert.pem --ca-cert=/etc/openvswitch/ca-cert.pem 2>&1 | tee /dev/$serialdev &\n";
+"${python}pox/pox.py openflow.of_01 --port=6634 --private-key=/etc/openvswitch/server-key.pem --certificate=/etc/openvswitch/server-cert.pem --ca-cert=/etc/openvswitch/ca-cert.pem 2>&1 | tee /dev/$serialdev &\n";
```

Comment:
I guess in this case POX needs an update too

Reply:
If you update `pox` then there is no need to stick to python 3.11.

Do you know, why we use pox here for testing? I mean, ovs/ovn comes with a very powerfull testsuite (https://github.com/ovn-org/ovn/blob/main/Documentation/topics/testing.rst).

On top, pox doesn't look quite active in the past 5 years (see: https://github.com/noxrepo/pox/tree/gar-experimental)

Reply:
> Do you know, why we use pox here for testing? 

Not really, and the person who added initially is long gone by now

Probably the test will have to be rewritten using something else to test the ssl + openvswitch

Reply:
yes, that's what I expect. But for now, I think it's still worth to go with python3.11 or fix pox in that regards, to get at least something running.